doc/cookbook/splitting.md

```diff
@@ -66,3 +66,146 @@ $ pachctl put-file users master -c -f user_data.json --split json --target-file-
 # bytes chunk into the split files.
 $ pachctl put-file users master -c -f user_data.txt --split line --target-file-bytes 100
 ```  
+
+## Specifying Header/Footer
+
+Additionally, if your data has a common header or footer, you can specify these
+manually via `pachctl put-header` or `pachctl put-footer`. This is helpful for CSV data.
+
+To do this, you'll need to specify the header/footer on the parent directory of your data. Below we have an example of splitting a CSV with a header, then setting the header explicitly. Notice that once we've set the header, whenever we get a file under that directory, the header is applied. You can still use glob patterns to get all the data under the directory, and in that case the header is still applied.
+
+```
+# Raw CSV
+$ cat users.csv 
+id,name,email
+4,alice,aaa@place.com
+7,bob,bbb@place.com
+# Take the raw CSV data minus the header and split it into multiple files:
+$ cat users.csv | tail -n +2 | pachctl put-file bar master users --split line
+Reading from stdin.
+$ pachctl list-file bar master
+NAME  TYPE SIZE 
+users dir  42B  
```

Comment:
Would it make sense to have a header/footer field in list-file on a dir that is a boolean to tell you if the dir has something embedded?